packages/iov-crypto/src/slip0010.ts

```diff
@@ -78,13 +81,31 @@ export class Slip0010 {
         // Step 1 of https://github.com/satoshilabs/slips/blob/master/slip-0010.md#private-parent-key--private-child-key
         // Calculate I = HMAC-SHA512(Key = c_par, Data = ser_P(point(k_par)) || ser_32(i)).
         // where the functions point() and ser_p() are defined in BIP-0032
-        throw new Error("Non-ed25519 normal key derivation not yet implemented");
+        const data = new Uint8Array([
```

Comment:
This doesn't seem too hard... once you figure out how it is done of course

Reply:
I was lucky there was a library doing the hard work. This hides probably 1-2 weeks of engeneering: `secp256k1.g.mul(p).encodeCompressed("hex")`